src/test/kotlin/org/rust/ide/lineMarkers/RsMethodLineSeparatorProviderTest.kt

```diff
@@ -27,14 +27,14 @@ class RsMethodLineSeparatorProviderTest : RsLineMarkerProviderTestBase() {
     fun `test trait`() {
         doTest("""
             trait Foo { // - Has implementations
-                const C1: i32 = 1;
-                fn f1();
-                fn f2();
-                fn f3() { // - null
+                const C1: i32 = 1; // - Has implementations
+                fn f1(); // - null,Has implementations
+                fn f2(); // - null,Has implementations
+                fn f3() { // - Has implementations
```

Comment:
I'm not really sure what is happening here and why the null has disappeared. I wonder if the comments can somehow affect the behaviour of `RsMethodLineSeparatorProvider`? Maybe we should strip the `// -` comments in the test?

Reply:
Hmm, maybe we should strip them. But also, and more reasonable, we should disable other line marker providers in these tests. Let's fix it later